cpp/models/abm/simulation.cpp

```diff
@@ -18,6 +18,9 @@
 * limitations under the License.
 */
 #include "abm/simulation.h"
+#include "memilio/utils/logging.h"
+#include "memilio/utils/random_number_generator.h"
+#include <random>
```

Comment:
I wrapped the #pragmas in preprocessor macros so they disappear when OpenMP is disabled to avoid this warning. Probably missed a #pragma somewhere. I think we talked about this.